CMakeLists.txt

```diff
@@ -192,26 +215,26 @@ set(MEMTEST_COMMAND
 separate_arguments(MEMTEST_COMMAND)
 
 foreach(TEST_SOURCE ${TEST_SOURCE_LIST})
-  cmake_path(GET TEST_SOURCE STEM TEST_TARGET)
+  cmake_path(GET TEST_SOURCE STEM TEST_STEM)
+  # Mirror the source directory for test collection and make unique target names
```

Comment:
The "make unique target names" part of this comment seems redundant with the comment on line 222.